release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -297,6 +297,12 @@ For more information, see xref:../nodes/jobs/nodes-nodes-jobs.adoc#nodes-nodes-j
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
+[id="ocp-4-12-auth-ccoctl-gcp-del-dir"]
+=== Credentials requests directory parameter for deleting GCP resources with the Cloud Credential Operator utility
+
+With this release, when you xref:../installing/installing_gcp/uninstalling-cluster-gcp.adoc#cco-ccoctl-deleting-sts-resources_uninstalling-cluster-gcp[delete GCP resources with the Cloud Credential Operator utility], you must specify the directory containing the files for the component `CredentialsRequest` objects.
```

Comment:
Just being nitpicky, but would the last phrase work as 

"the files for the `CredentialsRequest` component objects"

The way I am trying to read this is that you have to delete the "component objects." I think breaking up that phrase reads wordy, but I do not have the context of this feature, so I may be overworking it.

Reply:
I don't think this works - "`CredentialsRequest` objects" is a noun phrase that really shouldn't have things inserted between it. Thanks for thinking about it carefully, though :smiley: